### PR TITLE
Croud cluster bootstrap tutorial

### PR DESCRIPTION
**Pull request by matkuliak**
## Summary of the changes / Why this is an improvement

Croud cluster bootstrap tutorial

## Checklist

 - [x] [CLA](https://crate.io/community/contribute/cla/) is signed

### CODE REVIEW

File: docs/cluster-deployment/croud.rst [hunk 1]
```diff
@@ -0,0 +1,233 @@
...
+===========================
+
+In this tutorial, we will provide a step-by-step guide to deploying a cluster
+from scratch using our CLI application Croud. Our recommended procedure assumes
+you have completed the :ref:`signup process <sign-up>`. The simplest way to
+then deploy a cluster is to subscribe to CrateDB Cloud using your
+credit card. Here we will explain how to do so, from the initial subscribing to
+the deployment itself. Our payment processing and billing are powered by Stripe.
+
+This tutorial also assumes that you have our CLI tool Croud installed in your
```
Comment (kojinkai) [inline docs/cluster-deployment/croud.rst]: Is it necessary to refer to Croud as a CLI tool here? This was already explained in the above paragraph.
```suggestion
This tutorial also assumes that you have Croud installed in your
```

File: docs/cluster-deployment/croud.rst [hunk 2]
```diff
@@ -0,0 +1,233 @@
...
+system. In case not, follow the Croud `Getting started`_ section.
```
Comment (kojinkai) [inline docs/cluster-deployment/croud.rst]:
```suggestion
system. If you do not have Croud installed, follow the Croud `Getting started`_ section.
```

File: docs/cluster-deployment/croud.rst [hunk 3]
```diff
@@ -0,0 +1,233 @@
...
+.. contents::
+   :local:
+
+Log in
+======
+
+When working with Croud, the first step is always logging into your account.
+
+Croud, being a CLI application, is operated using commands. Commands should be
+supplemented following this format:
```
Comment (kojinkai) [inline docs/cluster-deployment/croud.rst]:
```suggestion
issued using the following format:
```

File: docs/cluster-deployment/croud.rst [hunk 4]
```diff
@@ -0,0 +1,233 @@
+.. _cluster-deployment-croud:
+
+
+===========================
+Deploy a cluster with Croud
+===========================
+
+In this tutorial, we will provide a step-by-step guide to deploying a cluster
+from scratch using our CLI application Croud. Our recommended procedure assumes
```
Comment (SStorm) [inline docs/cluster-deployment/croud.rst]: Maybe add a link to https://pypi.org/project/croud/?

```suggestion
from scratch using our CLI application - Croud. Our recommended procedure assumes
```

File: docs/cluster-deployment/croud.rst [hunk 5]
```diff
@@ -0,0 +1,233 @@
...
+you have completed the :ref:`signup process <sign-up>`. The simplest way to
+then deploy a cluster is to subscribe to CrateDB Cloud using your
+credit card. Here we will explain how to do so, from the initial subscribing to
+the deployment itself. Our payment processing and billing are powered by Stripe.
```
Comment (SStorm) [inline docs/cluster-deployment/croud.rst]: Worth mentioning that you can also use Azure and AWS?

File: docs/cluster-deployment/croud.rst [hunk 6]
```diff
@@ -0,0 +1,233 @@
...
+
+    sh$ croud projects create --name sampleproject
+
+This will create a new project named `sampleproject`. Check out full
+`documentation of projects in Croud`_.
+
+Create a subscription
+=====================
+
+To deploy a cluster, you need to create a subscription first. New subscription
```
Comment (SStorm) [inline docs/cluster-deployment/croud.rst]: Unfortunately, AFAIK the only time Subscriptions are actually created is when deploying a cluster - not enough to add a payment method on the billing page :/

File: docs/cluster-deployment/croud.rst [hunk 7]
```diff
@@ -0,0 +1,233 @@
...
+Scaling
+-------
+
+Required arguments:
+
+``--cluster-id``
+    The CrateDB cluster ID to use.
+
+``--unit``
+    The product scale unit to use.
```
Comment (SStorm) [inline docs/cluster-deployment/croud.rst]: The scale unit might be a bit confusing, worth explaining what it is?

File: docs/cluster-deployment/croud.rst [hunk 8]
```diff
@@ -0,0 +1,233 @@
...
+   ==> Info: Status: SENT (Your scaling request was sent to the region.)
+   ==> Info: Status: IN_PROGRESS (Scaling up from 2 to 3 nodes. Waiting for new node(s) to be present.)
+   ==> Success: Operation completed.
+   +--------------------------------------+------------------------+----------+
+   | id                                   | name                   | numNodes |
+   |--------------------------------------+------------------------+----------|
+   | 8d6a7c3c-61d5-11e9-a639-34e12d2331a1 | my-first-crate-cluster |        3 |
+   +--------------------------------------+------------------------+----------+
+
+Storage expansion
```
Comment (SStorm) [inline docs/cluster-deployment/croud.rst:193]: I would add an explanation here that expanding storage requires temporarily stopping the cluster.

File: docs/cluster-deployment/croud.rst [hunk 9]
```diff
@@ -0,0 +1,233 @@
...
+   ==> Info: Status: IN_PROGRESS (Starting cluster. Scaling back up to 3 nodes. Waiting for node(s) to be present.)
+   ==> Success: Operation completed.
+   +--------------------------------------+------------------------+------------------------------------+
+   | id                                   | name                   | hardware_specs                     |
+   |--------------------------------------+------------------------+------------------------------------|
+   | 8d6a7c3c-61d5-11e9-a639-34e12d2331a1 | my-first-crate-cluster | Disk size: 512.0 GiB               |
+   +--------------------------------------+------------------------+------------------------------------+
+
+.. note::
+    For all arguments available when scaling, see the `cluster scale`_ and
```
Comment (kojinkai) [inline docs/cluster-deployment/croud.rst]:
```suggestion
    For all available arguments for the scaling command, see the `cluster scale`_ and
```

**Review by kojinkai — APPROVED** [review]
Looks really good to me 👍🏼

**Pull request merged**